scripts/create_deprecated_modules.lean

```diff
@@ -0,0 +1,378 @@
+/-
+Copyright (c) 2025 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+
+--import Mathlib.Init
+import Mathlib.Tactic.Linter.DeprecatedModule
+import Std.Time.Zoned
+import Lean.Meta.Tactic.TryThis
+-- a comment here to test `keepTrailing
+
+/-!
+#  Create a deprecated module
+
+This file defines the lean script for creating a deprecated module.
+-/
+
+open Lean Elab Command
+
+namespace DeprecatedModule
+
+/--
+This file interacts with `git ...` quite a bit. `runCmd` takes as input the command-line
+function `git ...` and returns its stdout string as its output, although technically the
+command need not be `git ...`, this is the only situation where we use it.
+
+This is convenient to get both the output of the function, but also for reproducing the exact
+command-line text that produced the output for better reproducibility and error reporting.
+-/
+def runCmd (s : String) : IO String := do
+  let cmd::args := s.splitOn | EStateM.throw "Please provide at least one word in your command!"
```

Comment:
Careful: this does not handle quoting correctly. (It would parse `git commit -m "Multiple words"` as five arguments, not four, for instance.) This is fine for this file, but please add a big warning to the doc-string!